src/coreComponents/constitutive/fluid/multifluid/MultiFluidConstants.hpp

```diff
@@ -49,7 +49,7 @@ struct MultiFluidConstants
   /**
    * @brief Max number of SSI iterations
    */
-  static constexpr integer maxSSIIterations = 200;
+  static constexpr integer maxSSIIterations = 1000;
```

Comment:
make it an input parameter just in case?

Reply:
Agree. I will make these flash parameters xml accessible in a follow up PR.